### PR TITLE
add "send app link to device" button to featured experiment

### LINKED ISSUE #3542: The TxP install buttons from experiment details page no longer have an "Installing..." state other than the barberpole animation
**[Affected versions]:**
- Firefox 59.0.3 and above

**[Affected Platforms]:**
- All Windows
- All Mac OS
- All Linux

**[Prerequisites]:**
- Following 4 prefs have been set or added on a new profile: `xpinstall.signatures.required` = `false`, testpilot.env` = `dev`, `extensions.webapi.testing` = `true` and `extensions.install.requireBuiltInCerts` = `false`

**[Steps to reproduce]:**
1. Open Firefox browser and navigate to "testpilot.dev.mozaws.net page.
2. Click on on "Notes" experiment card and click on "Install Test Pilot & Enable Notes" button. 
3. While the doorhanger for install confirmation is displayed, observe the install button behavior.

**[Expected result]:**
- The green button string changes into "Installing..." with a barberpole animation behind.

**[Actual result]:**
- The green button gains the barberpole animation but the string remains unchanged.

**[Notes]**
- This issue also occurs for the "Install the Test Pilot Add-on" button when the experiment is already enabled.
- Here is a screen recording of the issue: https://goo.gl/feaGLC.

### LINKED ISSUE #3642: Add mobile button to featured experiment page
Here's the spec for mobile featured experiments.

https://mozilla.github.io/testpilot-assets/Test_Pilot_Website/01_Measurements/Mobile_Featured_Experiments/

Please DO NOT include the black ios button, just do the green "Send me the App" button. Also, Make the string  use the `mobileDialogLaunchButton` string.

Clicking this button should trigger the SMS/email modal.

### PR DESCRIPTION
**Pull request by meandavejustice**
- fixes #3642

featured-app-link

### CODE REVIEW

File: frontend/src/app/components/MobileDialog/index.js [hunk 1]
```diff
@@ -0,0 +1,284 @@
+// @flow
```
Comment (lmorchard) [inline frontend/src/app/components/MobileDialog/index.js]: Looks like you meant to move it into a shared component, but [there's still a copy of `MobileDialog.js` in the `app/containers/ExperimentContainer`](https://github.com/mozilla/testpilot/blob/5e01e25f701cb6fe1ff9c2b78229b89faf8df3cf/frontend/src/app/containers/ExperimentPage/MobileDialog.js) directory.

**Review by lmorchard — APPROVED** [review]
Except for the lingering copy of MobileDIalog.js, this looks okay to me. r+wc

**Comment by johngruen** [comment]
Thanks @meandavejustice! UX LGTM, feel free to merge once you fix les' issue

File: frontend/src/app/components/MobileDialog/index.js (hunk 1 above)
Comment (meandavejustice) [reply frontend/src/app/components/MobileDialog/index.js]: Oh whoops, removing now, thank you

**Comment by lmorchard** [comment]
Test failure looks related to general connectivity issues today. Trying a re-run

**Pull request merged**